### PR TITLE
Implemented the add-board command

### PR DESCRIPTION
**Pull request by ksk100**

### CODE REVIEW

File: src/commands/add-board.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  })
+  .callback(function (opts) {
+    logger.info("Adding new board");
+
+    // Build up arguments to send
+    var params = {
+      "name": opts.boardName,
+      "desc": opts.description,
+      "prefs_cardCovers" : opts.cardCoverImages ? "false" : "true",
+      "prefs_cardAging" : opts.cardAging ? "pirate" : "regular"
```
Comment (mheap) [inline src/commands/add-board.js]: I had to search the Trello docs to make sure "pirate" was the correct option!

**Comment by mheap** [comment]
This looks good. Perhaps it's worth running the refresh action for boards so that your new board is available straight away after running this command. Could you take a look at implementing that?

**Comment by ksk100** [comment]
Yeah good idea; I'll do that and resubmit it.

**Comment by ksk100** [comment]
All done

**Pull request merged**